plugins/annotations.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: default
+title: Annotations
+description: TinyMCE Annotations provides the ability to describe particular features or add general information to a piece of content and creates identifiers for each added annotation.
+keywords: annotation annotations
+---
+
+## Introduction
+The TinyMCE Annotation API provides the ability to add/modify/delete annotations, listen to text selection events, and retrieve all annotations with the same annotation name. The Annotations API is a part of the TinyMCE core and functions similar to the formatting APIs in TinyMCE core. 
+
+The primary value that the Annotations API provides is that it tags each annotation with a unique identifier(uid) which is accessible via `editor.annotator`. This highlights the annotated content and wraps it in annotation markers. These markers can either stay in the content or be removed on `getContent`, depending on the user configuration (persistent setting).
+
+## Using the Annotator Plugin
+
+To set up the TinyMCE Annotation plugin, perform the following procedure:
+
+### 1. Configure the Annotate Button
+
+       To configure the annotate button on your toolbar, make the following changes:
+
+       
+       ```js
+        setup: (ed) => {
+            ed.addButton('annotate-alpha', {
+              text: 'Annotate',
+              onclick: () => {
+                const comment = prompt('Comment with?');
+                ed.experimental.annotator.annotate('alpha', {
+                  comment
+                });
+                ed.focus();
+              },
+       ```
+> Note: The annotator plugin is still in its experimental stage, hence we are using 'experimental' in 'ed.experimental.annotator.annotate'. A user will see a *Using experimental API: annotator* warning on his console. Please ignore this warning, we are working on it.
+
+### 2. Registering the Annotator Plugin
+
+The annotator API supports multiple annotation functions. Each annotation function must be registered with the annotator `(editor.annotator)`. 
+
+```js
+ed.on('init', () => {
```

Comment:
Code indenting not consistent. I think we just need to indent this first line 4 spaces.